2015-03-01 14:57:28,698 - [INFO] - from play in pool-4-thread-1 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-03-01 17:08:11,681 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-03-01 17:08:51,464 - [ERROR] - from application in New I/O worker #2 


! @6lbgoah2j - Internal server error, for (POST) [/person] ->

play.PlayExceptions$CompilationException: Compilation error[identifier expected but 'type' found.]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-01 17:25:22,343 - [ERROR] - from application in New I/O worker #8 


! @6lbh0f0no - Internal server error, for (POST) [/person] ->

play.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : (String, String, String, enum.Gender.Gender) => models.Person
    (which expands to)  (String, String, String, enum.Gender.Value) => models.Person
 required: (String, String, enum.Gender.Gender) => ?
    (which expands to)  (String, String, enum.Gender.Value) => ?]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-01 17:27:20,864 - [ERROR] - from application in New I/O worker #8 


! @6lbh0lk6c - Internal server error, for (POST) [/person] ->

play.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : (String, String, String, enum.Gender.Gender) => models.Person
    (which expands to)  (String, String, String, enum.Gender.Value) => models.Person
 required: (String, String, enum.Gender.Gender) => ?
    (which expands to)  (String, String, enum.Gender.Value) => ?]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-01 17:29:06,804 - [ERROR] - from application in New I/O worker #8 


! @6lbh11kp1 - Internal server error, for (POST) [/person] ->

play.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : (String, String, String, enum.Gender.Gender) => models.Person
    (which expands to)  (String, String, String, enum.Gender.Value) => models.Person
 required: (String, String, enum.Gender.Gender) => ?
    (which expands to)  (String, String, enum.Gender.Value) => ?]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-01 17:53:35,520 - [ERROR] - from application in New I/O worker #2 


! @6lbh479g3 - Internal server error, for (POST) [/person] ->

play.PlayExceptions$CompilationException: Compilation error[Cannot find any HTTP Request here]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-01 17:53:56,626 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-03-01 17:54:36,524 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-03-01 17:54:36,741 - [ERROR] - from play in play-akka.actor.default-dispatcher-11 
Cannot invoke the action, eventually got an error: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'Person': was expecting ('true', 'false' or 'null')
 at [Source: java.io.StringReader@5922a772; line: 1, column: 7]

2015-03-01 17:54:36,748 - [ERROR] - from application in play-akka.actor.default-dispatcher-11 


! @6lbh4am51 - Internal server error, for (POST) [/person] ->

play.api.Application$$anon$1: Execution exception[[JsonParseException: Unrecognized token 'Person': was expecting ('true', 'false' or 'null')
 at [Source: java.io.StringReader@5922a772; line: 1, column: 7]]]
	at play.api.Application$class.handleError(Application.scala:296) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.DefaultApplication.handleError(Application.scala:402) [play_2.11-2.3.8.jar:2.3.8]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.8.jar:2.3.8]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.8.jar:2.3.8]
	at scala.Option.map(Option.scala:145) [scala-library-2.11.1.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:320) [play_2.11-2.3.8.jar:2.3.8]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:316) [play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library-2.11.1.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.11.1.jar:na]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46) [play-iteratees_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library-2.11.1.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library-2.11.1.jar:na]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41) [akka-actor_2.11-2.3.4.jar:na]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393) [akka-actor_2.11-2.3.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.11.1.jar:na]
Caused by: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'Person': was expecting ('true', 'false' or 'null')
 at [Source: java.io.StringReader@5922a772; line: 1, column: 7]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1524) ~[jackson-core-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:557) ~[jackson-core-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:2042) ~[jackson-core-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:1412) ~[jackson-core-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:679) ~[jackson-core-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:3031) ~[jackson-databind-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:2951) ~[jackson-databind-2.3.2.jar:2.3.2]
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:1587) ~[jackson-databind-2.3.2.jar:2.3.2]
	at play.api.libs.json.JacksonJson$.parseJsValue(JsValue.scala:486) ~[play-json_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.json.Json$.parse(Json.scala:19) ~[play-json_2.11-2.3.8.jar:2.3.8]
	at controllers.PersonController$$anonfun$create$1.apply(PersonController.scala:27) ~[classes/:na]
	at controllers.PersonController$$anonfun$create$1.apply(PersonController.scala:25) ~[classes/:na]
	at play.api.mvc.ActionBuilder$$anonfun$apply$16.apply(Action.scala:433) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.ActionBuilder$$anonfun$apply$16.apply(Action.scala:432) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$.invokeBlock(Action.scala:556) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$.invokeBlock(Action.scala:555) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.ActionBuilder$$anon$1.apply(Action.scala:518) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:129) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:128) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.Option.map(Option.scala:145) [scala-library-2.11.1.jar:na]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:128) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:121) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483) ~[play-iteratees_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519) ~[play-iteratees_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.8.jar:2.3.8]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496) ~[play-iteratees_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library-2.11.1.jar:na]
	... 6 common frames omitted

2015-03-01 17:55:44,507 - [ERROR] - from application in New I/O worker #2 


! @6lbh4eib5 - Internal server error, for (POST) [/person] ->

play.PlayExceptions$CompilationException: Compilation error[No Json serializer found for type models.Person. Try to implement an implicit Writes or Format for this type.]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-01 17:57:02,029 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

